Src/node_editor.py

```
@@ -50,26 +68,27 @@ def __init__(self, *args, **kwargs):
 
                 # Именно в группу задаём drop_callback, который создаёт ноду по перетягиванию её с окна справа
                 with dpg.group(tag="editor_group", drop_callback=self.drop_callback):
-                    
+
                     # Используем редактор нодов из DearPyGUI
                     with dpg.node_editor(tag="node_editor", callback=self.link_callback, \
                                         delink_callback=self.delink_callback, *args, **kwargs):
```

Ruff (0.15.21)

[warning] Star-arg unpacking after a keyword argument is strongly discouraged

(B026)